nvm.sh

```diff
@@ -1839,7 +1839,12 @@ nvm_get_arch() {
   elif [ "_${NVM_OS}" = "_aix" ]; then
     HOST_ARCH=ppc64
   else
-    HOST_ARCH="$(command uname -m)"
+    # If running MAC M1 :: arm64 ARCH and Darwin OS then use "x64" Architecture because node doesn't provide darwin_arm64 package
+    if [[ "_${NVM_OS}" == "_darwin" && "$(command uname -m)" == "arm64" ]]; then
```

Comment:
this is bash-specific syntax; nvm supports POSIX.
```suggestion
    if [ "_${NVM_OS}" = '_darwin' && "$(command uname -m)" = 'arm64' ]; then
```